src/OpenLoco/Config.h

```diff
@@ -170,7 +170,7 @@ namespace OpenLoco::Config
         int32_t autosave_amount = 12;
         bool showFPS = false;
         bool uncapFPS = false;
-        KeyboardShortcut shortcuts[35];
+        KeyboardShortcut shortcuts[44];
```

Comment:
Cant we access `ShortcutManager::count`

Reply:
I didn't want to include ShortcutManager.h here, as Config.h is included a lot. Should we? Or would you be fine moving the constant elsewhere, perhaps?

Reply:
Perhaps it's fine to just include ShortcutManager.h in Config.h, after all. The header isn't that large. Thoughts?

Reply:
yeah i think so. Other alternative would be to move it to limits.h but its not really a limit.